tokenList.json

```diff
@@ -50,7 +50,7 @@
         "totalSupply": 342015750
       }
     },
-    "image": "https://rawgit.com/CityOfZion/neo-tokens/master/assets/png/asa.png"
+    "image": "https://rawgit.com/undefined/neo-tokens/master/assets/png/asa.png"
```

Comment:
Looks like a bad merge happened here.  If it's easier, feel free to open a new PR with the file updated.

Reply:
Got it.